apps/challenges/views.py

```diff
@@ -53,7 +54,8 @@ def challenge_list(request, challenge_host_team_pk):
 @authentication_classes((TokenAuthentication,))
 def challenge_detail(request, challenge_host_team_pk, pk):
     try:
-        challenge_host_team = ChallengeHostTeam.objects.get(pk=challenge_host_team_pk)
+        challenge_host_team = ChallengeHostTeam.objects.get(
```

Comment:
same reason, if this is due to autopep8, then i will prefer that these should go in a seperate pr